app/code/Magento/ConfigurableProductGraphQl/etc/module.xml

```diff
@@ -12,6 +12,7 @@
             <module name="Magento_ConfigurableProduct"/>
             <module name="Magento_GraphQl"/>
             <module name="Magento_CatalogGraphQl"/>
+            <module name="Magento_QuoteGraphQl"/>
```

Comment:
It's necessary since `CartItemDetailsInput` field that is used in this module declared within a scope of `Magento_QuoteGraphQl`